src/views/ManageList.jsx

```diff
@@ -1,7 +1,90 @@
-export function ManageList() {
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
+
+export function ManageList({ listPath }) {
+	const [itemName, setItemName] = useState('');
+	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(0);
+
+	const handleChangeItem = (e) => {
+		setItemName(e.target.value);
+	};
+
+	const handleChangeDate = (e) => {
+		setDaysUntilNextPurchase(parseInt(e.target.value));
+	};
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		try {
+			const newItem = await addItem(listPath, {
```

Comment:
The const `newItem` is being set here but not used. Maybe remove to clean up the console.log?